pkg/kn/commands/service/configuration_edit_flags.go

```diff
@@ -117,6 +118,9 @@ func (p *ConfigurationEditFlags) addSharedFlags(command *cobra.Command) {
 	p.markFlagMakesRevision("max-scale")
 	command.Flags().StringVar(&p.AutoscaleWindow, "autoscale-window", "", "Duration to look back for making auto-scaling decisions. The service is scaled to zero if no request was received in during that time. (eg: 10s)")
 	p.markFlagMakesRevision("autoscale-window")
+	flags.AddBothBoolFlagsUnhidden(command.Flags(), &p.ClusterLocal, "cluster-local", "", false,
+		"specify that the service be private. (--no-cluster-local will make the service publicly available")
```

Comment:
Though the problem with using `Specify` here is that the auto-generated description for `--no-cluster-role` is:

```text
do not Specify that the service be private. (--no-cluster-local will make the service publicly available (default true)
```

which is not correct... Same for other `--no-xxx` flags like `--no-lock-to-digest` whose positive description starts with lower case. Created issue #636 to deal with this minor capitalization problem since could be done in code for all.